.devcontainer.json

```diff
@@ -51,40 +55,35 @@
     "python.pythonPath": "/usr/local/bin/python",
     "restructuredtext.linter.extraArgs": ["--max-line-length 80"],
     "[javascript]": {
-      "editor.formatOnSave": true,
+      "editor.formatOnSave": false,
```

Comment:
@jgerigmeyer why no more format on save?

Reply:
I was playing with `eslint.autoFixOnSave: true` above, which replaces this... but I'm not sure I like it. I'll probably switch it back.